lib/main.js

```diff
@@ -1,81 +1,191 @@
 'use babel'
 
-import {CompositeDisposable} from 'atom'
-import GoInformationView from './components/go-information-view'
-import {GoInformation} from './go-information'
-import {PanelManager} from './panel-manager'
-import {Formatter} from './formatter'
-import {Tester} from './tester'
-import TestPanel from './components/test-panel'
-import {TestPanelManager} from './test-panel-manager'
-import {ToolChecker} from './tool-checker'
+import {CompositeDisposable, Disposable} from 'atom'
+
+const oldPackages = [
+  'gofmt',
+  'tester-go',
+  'builder-go',
+  'autocomplete-go',
+  'godoc',
+  'gorename',
+  'go-hyperclick',
+  'navigator-go',
+  'go-get',
```

Comment:
I'm thinking `go-get` and `go-config` should not be in this list as they will still exist for other plugins (like `go-signature-statusbar` or `go-debug`)